Resources/Prototypes/WhiteDream/Roles/Antags/blood-cultist.yml

```
@@ -11,3 +11,15 @@
     back:
     - RitualDagger
     - RunedMetal20
+
+# Traitors
+- type: entity
+  parent: BaseMindRoleAntag
+  id: MindRoleBloodCultist
+  name: Blood Cultist Role
+#  description: mind-role-traitor-description
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)